src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs

```diff
@@ -1991,7 +1992,9 @@ private void ImportCall(ILOpcode opcode, int token)
             {
                 if (!_compilation.NodeFactory.TypeSystemContext.IsSpecialUnboxingThunkTargetMethod(canonMethod))
                 {
-                    hasHiddenParam = canonMethod.RequiresInstArg();
+                    hasHiddenParam = canonMethod.RequiresInstArg() || (canonMethod.IsArrayAddressMethod() &&
```

Comment:
```suggestion
                    hasHiddenParam = canonMethod.RequiresInstArg() || canonMethod.IsArrayAddressMethod();
```

To match 
https://github.com/dotnet/runtime/blob/b934bc02abe2bf03e8efd027a2d3bc14ba011acf/src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs#L492 ?

Reply:
Thanks, don't know why I added the extra conditions now, but works without them.